akka-docs/src/main/paradox/scala/guide/tutorial_1.md

```diff
@@ -78,7 +78,11 @@ convenient terminology, and we will stick to it.
 Creating a non-top-level actor is possible from any actor, by invoking `context.actorOf()` which has the exact same
 signature as its top-level counterpart. This is how it looks like in practice:
 
-@@snip [Hello.scala]($code$/scala/tutorial_1/ActorHierarchyExperiments.scala) { #print-refs }
+Scala
+:   @@snip [Hello.scala]($code$/scala/tutorial_1/ActorHierarchyExperiments.scala) { #print-refs }
```

Comment:
why do we use Hello.scala / Hello.java for all snippets? what is the purpose of that part?